llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -80,6 +80,7 @@
 # isort: off
 from llmfoundry.models.layers.fc import fcs  #  type: ignore
 from llmfoundry.models.utils.param_init_fns import generic_param_init_fn_  # type: ignore
+from llmfoundry.models.layers.norm import LPLayerNorm  # type: ignore
 # isort: on
```

Comment:
type ignore because the things that are imported are not actually used in this file, they are just imported so that hf will find the files it needs. isort so that this block remains isolated form the other normal imports